sqlalchemy_hana/requirements.py

```diff
@@ -259,3 +259,11 @@ def two_phase_recovery(self):
     @property
     def enforces_check_constraints(self):
         return exclusions.closed()
+
+    @property
+    def duplicate_key_raises_integrity_error(self):
+        """pyhdb raises the integrity_error, whereas hdbcli raises DBAPIError,
```

Comment:
This is actually a comment instead of a docstring. It describes a implementation detail and not the function/method. ;)